modules/library.js

```diff
@@ -0,0 +1,58 @@
+class Library {
+  constructor() {
+    this.books = [];
+    if (localStorage.getItem('books')) {
+      this.books = JSON.parse(localStorage.getItem('books'));
+    }
```

Comment:
- [ ] **[OPTIONAL]** You can do this in one line using a fallback value, like this:

 ```suggestion
    this.books = JSON.parse(localStorage.getItem('books')) || [];
    
      
    
```
In this case, if there's no `books` key in the local storage, the value of `this.books` will fallback to an empty array `[]`. 🤓 🔧

Reply:
Thank you, I knew it could be done, I just wasn't sure how to declare it. 🙏